torch/distributed/_tools/memory_tracker.py

```diff
@@ -176,7 +187,37 @@ def show_traces(self) -> None:
                 plt.plot(
                     [marker, marker], [min_val, max_val], "k-", lw=2, label=marker_name
                 )
-        plt.legend()
+
+    def save_stats(self, path: str) -> None:
+        """
+        Save the stats using pickle during runtime if users want to plot the traces
+        in other places like Bento.
```

Comment:
Does it matter if we mention Bento in open source?

Reply:
oh it is not open source..?